commit 2ba0935afc92b9288c01da246b3be53822277add
Author: Hui Zhou <hzhou321@anl.gov>
Date:   Sun Aug 14 23:29:09 2022 -0500

    UCS/DEBUG: replace PTR with void *
    
    The PTR macro is missing on the latest Arch linux.

diff --git src/ucs/debug/debug.c src/ucs/debug/debug.c
index b803636c2221..4bbf5095c419 100644
--- src/ucs/debug/debug.c
+++ src/ucs/debug/debug.c
@@ -272,10 +272,10 @@ static int load_file(struct backtrace_file *file)
         goto err_close;
     }
 
-    symcount = bfd_read_minisymbols(file->abfd, 0, (PTR)&file->syms, &size);
+    symcount = bfd_read_minisymbols(file->abfd, 0, (void *)&file->syms, &size);
     if (symcount == 0) {
         free(file->syms);
-        symcount = bfd_read_minisymbols(file->abfd, 1, (PTR)&file->syms, &size);
+        symcount = bfd_read_minisymbols(file->abfd, 1, (void *)&file->syms, &size);
     }
     if (symcount < 0) {
         goto err_close;
